.github/workflows/github-actions-deploy-dev.yml

```
@@ -8,20 +8,25 @@ on:
-    if: |
-      github.event_name == 'issue_comment' &&
-      github.event.issue.pull_request &&
-      contains(github.event.comment.body, '/deploy') &&
-      (github.event.comment.author_association == 'OWNER' || github.event.comment.author_association == 'MEMBER')
+    if: github.event_name == 'issue_comment'
     runs-on: ubuntu-24.04
     steps:
-      - name: Check deployment request
-        run: echo "Deployment requested via /deploy comment"
+      - name: Debug issue_comment
+        run: |
```

actionlint (1.7.12)

[error] "github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)